lib/tasks/remove-failed-followers.js

```diff
@@ -0,0 +1,58 @@
+/* jshint globalstrict:false, strict:false, sub: true */
+/* global print */
+exports.name = "remove-failed-followers";
+exports.group = "cleanup tasks";
+exports.args = [
+  {
+    name: "collection-integrity",
+    optional: false,
+    type: "jsonfile",
+    description: "json file created by analyze task"
+  }
+];
+exports.args_arangosh = " --server.endpoint AGENT-OR-COORDINATOR";
+exports.description = "Clears failed followers found by analyze task.";
+exports.selfTests = ["arango", "db", "leaderAgencyConnection"];
+exports.requires = "3.3.23 - 3.8.99";
+exports.info = `
+Removes failed followers found by the analyze task.
+`;
+
+exports.run = function (extra, args) {
+
+  // imports
+  const _ = require("lodash");
+  const helper = require('../helper.js');
+  const shards = helper.getValue("collection-integrity", args);
+
+  const trx = [];
+
+  for (const entry of shards.followerOnDeadServer) {
+    let path = `arango/Current/Collections/${entry.db}/${entry.name}/${entry.shard}/servers`;
+    let oper = {};
+    let prec = {};
+
+    oper[path] = _.without(entry.currentServers, entry.server);
```

Comment:
The order of elements is extremely important here, if we reorder the servers the leadership changes which will have undesired side effects.
The documentation of _.without does not guarantee that the order is preserved.
However It is implemented this way (in current version).

Do we want to take this risk?
Otherwise we can just use the Array.filter method, that is defined by the standard: https://262.ecma-international.org/5.1/#sec-15.4.4.20  as `... for each element in the array, in ascending order, ...` so the order is guaranteed.

A filter would look like this:
```suggestion
    oper[path] = entry.currentServers.filter(s => s !== entry.server);
```